client/components/icons.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+class Icons extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentIcon: 1
+    };
```

Comment:
Instead of using state to keep track of the current icon, pass a prop from your App into the Icons. Use that to choose the link coloring. Otherwise the back and forward buttons won't work.